portfolio/src/main/webapp/js/comments.js

```diff
@@ -57,18 +65,19 @@ function deleteComment(commentID) {
     params.append('action', 'delete');
     fetch('/comments', {method: 'POST', body: params}).then(response => {
         if (response.status === 200)
-            getComments(selectedSortType);
+            getComments(selectedSortType, selectedQuantityOfComments);
         else
             console.error(response.message);
     });
 }
 
-if (selectedSortType === 'date' || selectedSortType === 'rating')
-    getComments(selectedSortType);
-else {
-    getComments('date');
+if (selectedSortType !== 'date' && selectedSortType !== 'rating')
```

Comment:
Also, is this to select something by default? Can `selectedSortType` be something other than `date` or `rating`? If you were to add 5 more types of sorting, would you need to check if `selectedSortType` is different than all of them? Maybe we need something simpler here, like checking for null/undefined?